packages/api-gateway/src/drivers/http/routes/files/index.js

```diff
@@ -0,0 +1,53 @@
+const multer = require('fastify-multer'); // or import multer from 'fastify-multer'
+const boom = require('@hapi/boom');
+const { filesAdapter } = require('../../adapters');
+const { TIPO_EXTENSION } = require('../../utils/constants');
+
+const storage = multer.diskStorage({
+  filename(req, file, cb) {
+    cb(null, `${file.fieldname}-${Date.now()}`);
+  },
+});
+
+const fileFilter = (req, file, cb) => {
+  const tipoExtension = TIPO_EXTENSION.find((item) => item.mimeType === file.mimetype);
+  if (!tipoExtension) {
+    cb(null, () => {
+      throw boom.unsupportedMediaType(
+        '[files:uploadFile]: that media is not supported',
+      );
+    });
+  } else if (tipoExtension) {
+    cb(null, true);
+  }
+};
+
+const upload = multer({
+  storage,
+  limits: {
+    fileSize: 3 * 1024 * 1024,
```

Comment:
This should be storage like a constant in other file, so if we change our limit it will be easy update it